php/WP_Mock/Functions.php

```diff
@@ -105,12 +107,13 @@ public function register(string $function, array $args = [])
         /** @var Mockery\Mock $mock */
         $mock = $this->mockedFunctions[$function];
 
-        /** @var string $method */
+        /** @var callable-string $method */
         $method = preg_replace('/\\\\+/', '_', $function);
 
         $expectation = $this->setUpMock($mock, $method, $args);
 
-        Handler::register_handler($function, [$mock, $method]);
+        /** @phpstan-ignore-next-line */
+        Handler::registerHandler($function, [$mock, $method]);
```

Comment:
Is there a way to handle this phpstan without the ignore annotation?

Reply:
yeah actually there is, I have updated the PR